tests/py/previous_releases_test.py

```diff
@@ -0,0 +1,10 @@
+from qiskit_sphinx_theme.previous_releases import get_previous_versions_url
+
+
+def test_get_previous_versions_url() -> None:
```

Comment:
could you add a docstring please :) 

Also might be nice to parameterise this test? Its a small test so not a big deal if not, just a nice to have

Reply:
Also also, there seem to be much fewer test cases covered here compared to the whole test file above that got deleted. Is there a reason why?